### PR TITLE
feat: cache index data

### PR DESCRIPTION
**Pull request by alanshaw**
Caches satnav data in cloudflare's cache.

### CODE REVIEW

File: test/index.spec.js [hunk 1]
```diff
@@ -28,7 +28,8 @@ describe('freeway', () => {
       modules: true,
       r2Buckets: bucketNames,
       // r2Persist: true
-      kvNamespaces: ['BLOCKLY']
+      kvNamespaces: ['BLOCKLY'],
+      usageModel: 'unbound'
```
Comment (olizilla) [inline test/index.spec.js:32]: just to make test more like prod? or was it necessary for some wild reason?

**Review by olizilla — APPROVED** [review]
is it possible to test the cache behaviour? Like test a fetch, then delete from the test bucket, then fetch again and assert is retrieved? or some other way to prove that the cache is doing something?

File: test/index.spec.js (hunk 1 above)
Comment (alanshaw) [reply test/index.spec.js:32]: In one of the tests we read 65 indexes, and with bound usage it took us over the limit of 50.

**Review by vasco-santos — APPROVED** [review]

File: test/index.spec.js (hunk 1 above)
Comment (alanshaw) [reply test/index.spec.js:32]: It also makes it more like prod ;)

**Pull request merged**